packages/tools/kadena-cli/src/services/index.ts

```diff
@@ -0,0 +1,17 @@
+import { memoryFileSystemService } from './fs.memory.service.js';
+import type { IFileSystemService } from './fs.service.js';
+import { fileSystemService } from './fs.service.js';
+export {
+  getCallHistory,
+  serviceCalledWith as mockServiceCalledWith,
```

Comment:
is this still needed?

Reply:
Looks like this `as` was typescript trying to help me in refactoring the name, doesn't need to be aliased